packages/react/src/app-shell/AppShell.tsx

```diff
@@ -0,0 +1,42 @@
+import { type ReactNode, forwardRef } from "react";
+
+import { Box, type BoxProps } from "../box";
+import { Flex } from "../flex";
+import { extractSprinkles } from "../sprinkles";
+import * as styles from "./AppShell.css";
+
+type AppShellProps = BoxProps<
+  "div",
+  {
+    header?: ReactNode;
+    navbar?: ReactNode;
+  }
+>;
+
+export const AppShell = forwardRef<HTMLDivElement, AppShellProps>(
+  ({ children, className, header, navbar, ...props }, ref) => {
+    const { restProps, sprinkleProps } = extractSprinkles(props);
+
+    return (
+      <Flex
+        ref={ref}
+        {...styles.appShell({}, className)}
+        {...sprinkleProps}
+        {...restProps}
+      >
+        {header}
+        <Flex
+          alignItems="start"
+          flexDirection="row"
+          gap="0"
+          justifyContent="start"
+        >
+          <Flex>{navbar}</Flex>
```

Comment:
Do we need Flex here, I think Box is the most suitable.

Reply:
I think, we can just skip `Box` or `Flex` and just leave {sidenav} to as it is

Reply:
That's more preferred I believe if we are not appending any style.